coverage/clover.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Dont commit coverage folder.
Add it to .gitignore

Reply:
noted

Reply:
Why coverage folder is not removed ? 
THis should not be merged.